rpcs3/rpcs3qt/log_frame.cpp

```diff
@@ -304,6 +310,7 @@ void log_frame::CreateAndConnectActions()
 	{
 		QMenu* menu = m_tty->createStandardContextMenu();
 		menu->addAction(m_clearTTYAct);
+		menu->addAction(m_stackAct_tty);
```

Comment:
add a separator before this line

Reply:
I don't know why it doesn't say outdated, but the new version is here:
https://github.com/RPCS3/rpcs3/pull/6542/commits/75498bac7b8678a95e2c28e5dac34d9ead04baf8

So we seem to be good.